pytensor/tensor/nlinalg.py

```diff
@@ -112,6 +95,8 @@ class MatrixInverse(Op):
     """
 
     __props__ = ()
+    gufunc_signature = "(m,m)->(m,m)"
+    gufunc_spec = ("numpy.linalg.inv", 1, 1)
```

Comment:
what are `1, 1`? inputs and outputs?

Reply:
It's the number of inputs and outputs, just following the same convetion of `nfunc_spec` for Scalar Ops